docs/content/docs/5.v9/3.options/10.misc.md

```diff
@@ -18,6 +18,9 @@ About how to define the locale detector, see the [`defineI18nLocaleDetector` API
 - `autoImportTranslationFunctions` (default: `false`) - Automatically imports/initializes `$t`, `$rt`, `$d`, `$n`, `$tm` and `$te` functions in `<script setup>` when used.
 ::callout{icon="i-heroicons-exclamation-triangle" color="amber"}
 This feature relies on [Nuxt's Auto-imports](https://nuxt.com/docs/guide/concepts/auto-imports) and will not work if this has been disabled.
+- `typedPages` (default: `false`) - Generates route types used in composables and configuration.
+::callout{icon="i-heroicons-exclamation-triangle" color="amber"}
+This feature relies on [Nuxt's `experimental.typedRoutes`](https://nuxt.com/docs/guide/going-further/experimental-features#typedpages) and will not work if this is not enabled.
```

Comment:
Maybe its possible to enable the `experimental.typedRoutes` for users, when they set `typedPages` to `true`? :)

Reply:
Sounds good to me, since it's an experimental feature in Nuxt too we might as well automatically enable this when Nuxt's `experimental.typedPages` is enabled and when i18n's `experimental.typedPages` is left unset. Explicitly setting the property to false should still disable it.